contributors/devel/pull-requests.md

```diff
@@ -1,111 +1,326 @@
 <!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Pull Request Process - Who is this for?](#pull-request-process---who-is-this-for-)
+- [Before You Submit a PR - Prerequisites](#before-you-submit-a-pr---prerequisites)
+  * [Run Local Verifications](#run-local-verifications)
+  * [Sign the CLA](#sign-the-cla)
+- [What to Expect When You Submit a PR](#what-to-expect-when-you-submit-a-pr)
+  * [Update the Release Note Label (and Write Release Notes if Needed)](#update-the-release-note-label--and-write-release-notes-if-needed-)
+  * [The Testing and Merge Workflow](#the-testing-and-merge-workflow)
+  * [Comment Commands Reference](#comment-commands-reference)
+  * [Automation](#automation)
+  * [How the e2e Tests Work](#how-the-e2e-tests-work)
+- [Why was my PR closed?](#why-was-my-pr-closed-)
+- [Why is my PR not getting reviewed?](#why-is-my-pr-not-getting-reviewed-)
+- [Best Practices for Faster Reviews](#best-practices-for-faster-reviews)
+  * [0. Familiarize yourself with project conventions](#0-familiarize-yourself-with-project-conventions)
+  * [1. Is the feature wanted? Make a Design Doc or Sketch PR](#1-is-the-feature-wanted--make-a-design-doc-or-sketch-pr)
+  * [2. Smaller Is Better: Small Commits, Small PRs](#2-smaller-is-better--small-commits--small-prs)
+  * [3. Open a Different PR for Fixes and Generic Features](#3-open-a-different-pr-for-fixes-and-generic-features)
+  * [4. Comments Matter](#4-comments-matter)
+  * [5. Test](#5-test)
+  * [6. Squashing and Commit Titles](#6-squashing-and-commit-titles)
+  * [7. KISS, YAGNI, MVP, etc.](#7-kiss--yagni--mvp--etc)
+  * [8. It's OK to Push Back](#8-it-s-ok-to-push-back)
+  * [Common Sense and Courtesy](#common-sense-and-courtesy)
+<!-- END MUNGE: GENERATED_TOC -->
 
-- [Pull Request Process](#pull-request-process)
-- [Life of a Pull Request](#life-of-a-pull-request)
-  - [Before sending a pull request](#before-sending-a-pull-request)
-  - [Release Notes](#release-notes)
-    - [Reviewing pre-release notes](#reviewing-pre-release-notes)
-  - [Visual overview](#visual-overview)
-- [Other notes](#other-notes)
-- [Automation](#automation)
+# Pull Request Process - Who is this for?
 
-<!-- END MUNGE: GENERATED_TOC -->
+This doc explains the process and best practices for submitting a PR to the [Kubernetes project](https://github.com/kubernetes/kubernetes). It should serve as a reference for all contributors, and be useful especially to new and infrequent submitters.
+
+# Before You Submit a PR - Prerequisites
+
+If you're just getting started, please refer to the [development guide](development.md) to set up your local dev environment and get coding. This guide is for contributors who already have a PR to submit.
```

Comment:
Flip the order of this sentence:  This guide is for contributors who already have a PR to submit.  If you're looking for information on setting up your developer environment and creating code to contribute to Kubernetes, see the [development guide](development.md).